### ISSUE TITLE
Can't bind key with controller 

### ISSUE THREAD

**Issue by thebear45800** [NitrOP7674/FreeplayCheckpoint#35]
Hello, I just install the plugin, but I can't set any key to command. 
Does the plugin need update since last RocketLeague update ?

**Comment by NitrOP7674** [comment]
No, everything is working fine with the plugin after the last RL update.  If you can provide more info and screenshots I can try to help.

**Comment by thebear45800** [comment]
Can you add me on discord ?
I can't because your profile doesn't accept friend request

My username : t.tv/thebear45#4640

My issue is I can't bind any key if I clic on blue button and after press button on controller nothing happens.

![image](https://user-images.githubusercontent.com/26141460/225438289-57e73149-7e5b-44b7-9808-f9b2c7df52fd.png)

**Comment by thebear45800** [comment]
Because I really want use this plugin I reinstall bakkesmod from scratch, and all is work fine now :)
Sorry for disturbate

**Comment by NitrOP7674** [comment]
No worries, glad you were able to fix it.  FWIW the way the binding buttons work is you hold the button on the controller you want to assign to the action while you click the button for it in the menu.  Or you can just assign the commands in the "Bindings" tab if you prefer.

Also FWIW you don't need to be friends with someone to send them a message.

**Comment by thebear45800** [comment]
I am curious to know how to DM someone withou being friend, because when I go to "Direct message" I see that : 
![image](https://user-images.githubusercontent.com/26141460/225559528-f151855d-0c15-4692-ba95-95c720285bc9.png)

Or if I click to "Find or start a conversation" I see that : 
![image](https://user-images.githubusercontent.com/26141460/225559797-2099b32a-3229-4e5f-bf38-bd7df5337306.png)

But it search only on my list friend/servers ...

**Comment by NitrOP7674** [comment]
Oh strange..you must be right. I get dms from people I don't know all the time even though we're not friends, but it's because we share a server. I don't see how to allow dms from anyone but it seems there's no privacy anyway once you're on a couple big servers so IDK why not..

I guess I'll make a little discord server then.

https://discord.gg/SPBxrtfrZw
